### ISSUE TITLE
[VL] [BUILD] ERROR: cannot verify repo.anaconda.com's certificate

### ISSUE THREAD

**Issue by ychris78** [apache/incubator-gluten#5584]
### Description

When I compile gluten on Alibaba Cloud ECS according to **_docs/developers/docker_centos7.md_** in the project, the following error always occurs. 

**Error log:**

```
+ run_and_time install_conda
+ install_conda
+ cd /tmp/velox-deps
+ mkdir -p conda
+ cd conda
+ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
--2024-04-29 06:40:20--  https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
Resolving repo.anaconda.com (repo.anaconda.com)... 104.16.191.158, 104.16.32.241, 2606:4700::6810:bf9e, ...
Connecting to repo.anaconda.com (repo.anaconda.com)|104.16.191.158|:443... connected.
ERROR: cannot verify repo.anaconda.com's certificate, issued by '/C=US/O=Let\'s Encrypt/CN=E1':
  Issued certificate has expired.
To connect to repo.anaconda.com insecurely, use `--no-check-certificate'.

```

**Comment by Yohahaha** [comment]
> When I compile gluten on Alibaba Cloud ECS

Are you EMR user or self-build user?

**Comment by PHILO-HE** [comment]
@ychris78, you can optionally add `--no-check-certificate` for wget.

BTW, we have a nightly build job that runs on pure centos/ubuntu OS. See https://github.com/apache/incubator-gluten/blob/main/.github/workflows/velox_nightly.yml for reference.

**Comment by ychris78** [comment]
> @ychris78, you can optionally add `--no-check-certificate` for wget.
> 
> BTW, we have a nightly build job that runs on pure centos/ubuntu OS. See https://github.com/apache/incubator-gluten/blob/main/.github/workflows/velox_nightly.yml for reference.

The issue has been fixed in #5566, thanks

**Comment by ychris78** [comment]
The issue has been fixed in #5566, thanks

**Comment by PHILO-HE** [comment]
Let's close this issue. Thanks!
